ajax/libs/vivagraphjs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "vivagraphjs",
+  "version": "0.8.2",
+  "description": "Graph Drawing Library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anvaka/VivaGraphJS.git"
+  },
+  "keywords": [
+    "graph"
+  ],
+  "author": "Andrei Kashcha",
+  "license": "BSD-3-Clause",
+  "npmName": "vivagraphjs",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "vivagraph.min.js"
```

Comment:
We usually put the important info upper in `package.json`, so please move `filename` field prior.